test/e2e/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -x
+set -euo pipefail
+
+CWD=$(dirname "$0")
+CLI=ff
+STACK_DIR=~/.firefly/stacks
+STACK_NAME=firefly-e2e
+STACK_FILE=$STACK_DIR/$STACK_NAME/stack.json
+DOWNLOAD_CLI=true
+CREATE_STACK=true
+
+if $DOWNLOAD_CLI
+then
+	go get github.com/kaleido-io/firefly-cli/ff
```

Comment:
How where does this binary go, if I do a `go get` like this from within a sub-directory inside of a module? How would I find it to execute it?

Reply:
I believe `go get` installs to `~/go/bin`

Reply:
So this may depend on you having that on your `PATH`...

Reply:
Yeah, that was what I was thinking. I believe the `Makefile` uses some go tools... will go take a look at how I worked it out there

Reply:
On a further look, `go get` may have the unintended consequence of adding a dependency to `go.mod`
https://stackoverflow.com/a/52764550/638122

Reply:
Seems the dependency issue can be worked around with `GO111MODULE=off` to perform more of a "global" install